src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py

```diff
@@ -111,6 +110,7 @@ def create(self, size, isolate_nspace, pool, mode, uid, gid):
             # persist subvolume metadata
             qpath = subvol_path.decode('utf-8')
             self.init_config(SubvolumeV1.VERSION, subvolume_type, qpath, initial_state)
+            self.mark_subvolume()
```

Comment:
@mchangir should we have commit f82b2a65935fe1c1d030ca068d7dc39b343b328a as a separate PR since we might want to backport that to earlier releases quickly?

Reply:
Before v2 subvolumes were introduced, the `ceph.dir.subvolume` xattr management did not exist in the system, and so the function call to `self.mark_subvolume()` was not necessary as well.

Also, with the introduction of v2 subvolumes, I doubt anybody in the field would go out of the way to create v1 subvolumes.

So, the commit f82b2a6 need not be forked out into a separate PR. The v1 subvolume code is just a repository of common methods used by v1 and v2 subvol management code. The v1 subvol management code is effectively unreachable directly by end users since the `self.mark_subvolume()` is part of the `def create()` method which is subvol version specific and there's no way to request creation of a v1 subvol from the CLI.

Reply:
That makes sense. Thx for explaining @mchangir.

cc @kotreshhr too.